catalog/capabilities/database/info.json

```diff
@@ -11,14 +11,8 @@
       "required": true,
```

Comment:
Could we make it not required (since there is a default value anyway)?

Reply:
The required state here means "you cannot leave this field empty". This in contrast to a field with a default value but where passing an empty value is allowed too. This is because the CLI and forms don't have a way to distinguish between empty values and non-existing ones.